examples/simple-filter-xml/res/configuration.toml

```diff
@@ -51,3 +51,31 @@ File = './logs/simple-filter-xml.log'
 ApplicationName = "simple-filter-xml"
 
 
+
+[Pipeline]
```

Comment:
Why add this here? This example is a custom function pipeline.

Reply:
this was part of design phase, before we had everything